packages/js-drive/lib/validator/ValidatorSet.js

```diff
@@ -133,9 +136,13 @@ class ValidatorSet {
     const isThisNodeMember = !!quorumMembers
       .find((member) => member.valid && member.proTxHash === proTxHash);
 
-    this.validators = quorumMembers
+    this.validators = await Promise.all(quorumMembers
       .filter((member) => member.valid)
-      .map((member) => Validator.createFromQuorumMember(member, isThisNodeMember));
+      .map(async (member) => {
+        const networkInfo = await this.fetchNetworkInfo(member);
```

Comment:
You don't need to fetch it, you already have it in SML https://github.com/dashevo/dashcore-lib/blob/4ac997530397494fd16f20e87fa1f36e0c3db227/lib/deterministicmnlist/SimplifiedMNListEntry.js#L83